src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -2423,7 +2423,7 @@ static void sub_6CBCE2(
     gMapSizeUnits = 255 * 32;
     gMapSizeMinus2 = (255 * 32) + 286;
     gMapSize = 256;
-    gMapSizeMaxXY = (256 * 32) - 1;
+    gMapSizeMaxXY = (MAXIMUM_MAP_SIZE_TECHNICAL * 32) - 1;
```

Comment:
You can also apply `MAXIMUM_MAP_SIZE_TECHNICAL` (or `MAXIMUM_MAP_SIZE_TECHNICAL - 1`) to the lines above.

Reply:
Done